bazel/linux/runner.bzl

```diff
@@ -218,6 +231,8 @@ def create_runner(ctx, archs, code, runfiles = None, extra = {}):
         post = "cd {dest}; cp -L %s ./init.sh" % (shell.quote(init.short_path)),
     )
     outside_runfiles = torun.outside_runfiles.merge(ctx.runfiles([runtime_root, ki.image]))
+    if ctx.attr.bootram_image:
+       outside_runfiles = outside_runfiles.merge(ctx.runfiles([bootram_info.image]))
```

Comment:
it seems a little dirty to check for one variable (`ctx.attr.bootram_image`) and then use a value from a different variable (`bootram_info.image`).  maybe you can move the `ki.arch == aarch64` test down here?

Reply:
Yeah.  Rearanged the code a little.